app/services/uphold/models/refreshment.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'uri'
+
+module Uphold
+  module Models
+    class Refreshment < Client
```

Comment:
Would you clarify naming, e.g. rename `def refresh` to `def refresh_authorization` and rename `Refreshment` to `Authorization`?

Reply:
Done.